src/blend.hpp

```diff
@@ -35,6 +38,8 @@ struct BImage {
     int width, height;
     Tinter tint;
     std::unique_ptr<mapnik::image_rgba8> im_ptr;
+    mapnik::image_rgba8 * im_raw_ptr;
```

Comment:
I'm having a hard to understanding why `im_raw_ptr;` is needed. Maybe it is, but my brain is wondering if you can make it work without it just from simplicity?

Reply:
The `Mapnik.Image` object owns the `mapnik::image_any` object containing a `mapnik::image_rgba8` -- so I can use the existing `std::unique_ptr` when I am trying to get information from within it as I don't want to transfer ownership of the image. Therefore, to make it work for both, I just use a raw ptr now as well.

Reply:
OKay, merge if you are satisfied.